healthcare/api-client/v1/fhir/fhir_stores_test.py

```diff
@@ -0,0 +1,255 @@
+# Copyright 2018 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import uuid
+
+import backoff
+import pytest
+from googleapiclient.errors import HttpError
+from google.cloud import storage
+
+# Add datasets for bootstrapping datasets for testing
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", "datasets"))  # noqa
+import datasets
+import fhir_stores
+
+cloud_region = "us-central1"
+project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
+
+dataset_id = "test_dataset_{}".format(uuid.uuid4())
+fhir_store_id = "test_fhir_store-{}".format(uuid.uuid4())
+
+gcs_uri = os.environ["CLOUD_STORAGE_BUCKET"]
+RESOURCES = os.path.join(os.path.dirname(__file__), "resources")
+source_file_name = "Patient.json"
+resource_file = os.path.join(RESOURCES, source_file_name)
+import_object = "{}/{}".format(gcs_uri, source_file_name)
+
+
+@pytest.fixture(scope="module")
+def test_dataset():
+    @backoff.on_exception(backoff.expo, HttpError, max_time=60)
+    def create():
+        try:
+            datasets.create_dataset(project_id, cloud_region, dataset_id)
+        except HttpError as err:
+            # We ignore 409 conflict here, because we know it's most
+            # likely the first request failed on the client side, but
+            # the creation suceeded on the server side.
+            if err.resp.status == 409:
+                print("Got exception {} while creating dataset".format(err.resp.status))
+            else:
+                raise
+
+    create()
+
+    yield
+
+    # Clean up
+    @backoff.on_exception(backoff.expo, HttpError, max_time=60)
+    def clean_up():
+        try:
+            datasets.delete_dataset(project_id, cloud_region, dataset_id)
+        except HttpError as err:
+            # The API returns 403 when the dataset doesn't exist.
```

Comment:
Same as above (and a few other places - I won't keep pointing it out)

Reply:
Yes, black is requiring it.